### DISCUSSION TITLE
Akku per Huawei Netzteil laden

### DISCUSSION THREAD

**Discussion by mastertape** [noone2k/hm2500pub#6] [Ideas]
Hat hier schon jemand erfahrungen gemacht, diesen Akku mithilfe eines Huaweinetzteils zu laden ( was auch von opendtu untertützt wird)?

Wie geht ihr dabei vor?
Was benutzt Ihr an Soft & Hardware...
Und gibt es evtl. auch jemanden der dies erfolgreich mit 2 Akkus dieses betreibt?

Ich habe 2 Akkus und ein 3000 Watt per CAN-Bus angestuerwerden kann schon hier rumliegen und würde gerne dieses Nutzen um mit der Leistung von 14 Panels effektiver potentiellen "ÜBerschuss" nutzen können ;)

**Comment by jn30** [comment]
Kannst auch 3 Akkus laden mit;) Das r4850 macht 3kW😂

Ansonsten hast quasi alles schon beschrieben. ESP32 mit Can-Adapter und gib ihm.

Nutze HomeAssistant als übergeordnete Steuerung.

**Reply by mastertape** [reply]
Vom rein rechnischen Wert leuchtet mir das soweit auch ein... Naja ich habe angst wegen der evtl. gebrückten Massen... wie verhält sich das ohne mppt tracker und so...

Welche Software nutzt du zur steuerung des Ladegerät (Opendtu oder eines der anderen Github Scripte? 
Was sind die einstellungen mit denen du gute erfahrungen gemacht hast?

Wäre super neugierig

**Reply by jn30** [reply]
Naja in den Batterien sind Tracker drin. 

Aufbau is bei mir:

Huawei > Y-Kabel MC4 > IN1/2 Batterie > HM600

AC Sicherung 10A vorm Huawei
DC Sicherung 20A zwischen Huawei und Batterie 
Softwareseitig das Netzteil auf max 19A abgeriegelt.

Gibt n ESP-HomeProjekt für das Netzteil. Das hab ich auf meine Bedürfnisse angepasst.

**Reply by mastertape** [reply]
Ich habe da mehre Can Bus Projekte gefunden... und die möglichkeit es per opendtu zu regeln.. Meinst Du das hier?

https://github.com/KlausLi/Esp-HuaweiR4850-Controller

Irgendwie sagt mir mein Bauchgefühl das es keine gute idee ist einfach beide akkus gleichzeitig am netzteil laufen zu lassen...
Vielleicht ist es aber auch unbegründet. Jedenfalls gab es einmal sehr komische effekte als ich beide akkus gleichzeitig am hoymiles hms-1600-4t hattte

**Reply by jn30** [reply]
https://github.com/mb-software/esphome-huawei-r4850

hab das da im Einsatz=)

**Reply by mastertape** [reply]
noch mal ne frage  - welchen can controller nutzt du?

**Reply by jn30** [reply]
Der der beschrieben is:) kannst dir die genaue Bezeichnung aus der Readme des Projekts ziehen....
